impeller/playground/backend/metal/playground_impl_mtl.mm

```diff
@@ -34,23 +35,25 @@
 
 static std::vector<std::shared_ptr<fml::Mapping>>
 ShaderLibraryMappingsForPlayground() {
-  return {
-      std::make_shared<fml::NonOwnedMapping>(impeller_entity_shaders_data,
-                                             impeller_entity_shaders_length),
-      std::make_shared<fml::NonOwnedMapping>(impeller_modern_shaders_data,
-                                             impeller_modern_shaders_length),
-      std::make_shared<fml::NonOwnedMapping>(
-          impeller_framebuffer_blend_shaders_data,
-          impeller_framebuffer_blend_shaders_length),
-      std::make_shared<fml::NonOwnedMapping>(impeller_fixtures_shaders_data,
-                                             impeller_fixtures_shaders_length),
-      std::make_shared<fml::NonOwnedMapping>(
-          impeller_subgroup_fixtures_shaders_data,
-          impeller_subgroup_fixtures_shaders_length),
-      std::make_shared<fml::NonOwnedMapping>(impeller_imgui_shaders_data,
-                                             impeller_imgui_shaders_length),
-      std::make_shared<fml::NonOwnedMapping>(impeller_scene_shaders_data,
-                                             impeller_scene_shaders_length),
+  return {std::make_shared<fml::NonOwnedMapping>(
+              impeller_entity_shaders_data, impeller_entity_shaders_length),
+          std::make_shared<fml::NonOwnedMapping>(
+              impeller_modern_shaders_data, impeller_modern_shaders_length),
+          std::make_shared<fml::NonOwnedMapping>(
+              impeller_framebuffer_blend_shaders_data,
+              impeller_framebuffer_blend_shaders_length),
+          std::make_shared<fml::NonOwnedMapping>(
+              impeller_fixtures_shaders_data, impeller_fixtures_shaders_length),
+          std::make_shared<fml::NonOwnedMapping>(
+              impeller_subgroup_fixtures_shaders_data,
+              impeller_subgroup_fixtures_shaders_length),
+          std::make_shared<fml::NonOwnedMapping>(impeller_imgui_shaders_data,
+                                                 impeller_imgui_shaders_length),
+          std::make_shared<fml::NonOwnedMapping>(impeller_scene_shaders_data,
+                                                 impeller_scene_shaders_length),
+          std::make_shared<fml::NonOwnedMapping>(
+              impeller_tessellation_shaders_data,
```

Comment:
Nit: Consider naming these something other than tessellation shaders since these don't setup tessellation pipeline stages in the rasterizer.

Reply:
done